pilot/pkg/serviceregistry/kube/controller/controller_test.go

```diff
@@ -498,6 +498,76 @@ func TestGetProxyServiceInstances(t *testing.T) {
 	}
 }
 
+func TestGetProxyServiceInstancesWithMultiIPs(t *testing.T) {
+	controller, fx := newFakeController(t)
```

Comment:
Add more test cases, at least including :

- multi ip-> one service:port

- multi ip -> multi ports

- single ip -> single port

- single ip -> multi port

Reply:
OK, will ping you after added. Thanks a lot

Reply:
@hzxuzhonghu  Already modified. Any other suggestion just ping me.